### PR TITLE
ES Schema > Add instructors to Event

### PR DESCRIPTION
**Pull request by luannguyengo1**

### CODE REVIEW

**Comment by mrsinguyen** [comment]
cc @drushman

File: es/Schema.php [hunk 1]
```diff
@@ -946,6 +946,10 @@ class Schema
         'location_name'            => ['type' => self::T_KEYWORD] + self::ANALYZED,
         'module_title'             => ['type' => self::T_KEYWORD] + self::ANALYZED,
         'instructor_ids'           => ['type' => self::T_INT],
+        'instructors'              => [
+                'type'       => self::T_NESTED,
+                'properties' => self::ACCOUNT_MAPPING['properties'],
```
Comment (drushman) [inline es/Schema.php]: Should we index major property on account only? `self::ACCOUNT_MAPPING['properties']` seems really complext and don't think we need all of them in event scenario

Comment (mrsinguyen) [reply es/Schema.php]: Agree, let check with the frontend dev to make sure what's information is needed in report.

**Comment by mrsinguyen** [comment]
looks good to me @drushman

**Pull request merged**